qutebrowser/javascript/caret.js

```diff
@@ -859,12 +859,25 @@ window._qutebrowser.caret = (function() {
         const style = ".CaretBrowsing_Caret {" +
             "  position: absolute;" +
             "  z-index: 2147483647;" +
-            "  min-height: 10px;" +
-            "  background-color: #000;" +
+            "  min-height: 1em;" +
+            "  min-width: 0.2em;" +
+            "  --inherited-color: inherit;" +
+            "  background-color: var(--inherited-color, #000);" +
+            "  color: var(--inherited-color, #000);" +
+            "  mix-blend-mode: difference;" +
+            "  filter: invert(85%);" +
+            "  animation: blink 1s step-end infinite;" +
+            "}";
+        const blink = "@keyframes blink {" +
+            "50% { visibility: hidden; }" +
             "}";
         const node = document.createElement("style");
         node.innerHTML = style;
         document.body.appendChild(node);
+
+        const node2 = document.createElement("style");
+        node2.innerHTML = blink;
+        document.body.appendChild(node2);
```

Comment:
I still don't understand why a second element is needed here. I tried this and it seems to work just fine:

```diff
diff --git a/qutebrowser/javascript/caret.js b/qutebrowser/javascript/caret.js
index 800a27735..3a4272984 100644
--- a/qutebrowser/javascript/caret.js
+++ b/qutebrowser/javascript/caret.js
@@ -867,17 +867,13 @@ window._qutebrowser.caret = (function() {
             "  mix-blend-mode: difference;" +
             "  filter: invert(85%);" +
             "  animation: blink 1s step-end infinite;" +
-            "}";
-        const blink = "@keyframes blink {" +
-            "50% { visibility: hidden; }" +
+            "}" +
+            "@keyframes blink {" +
+            "  50% { visibility: hidden; }" +
             "}";
         const node = document.createElement("style");
         node.innerHTML = style;
         document.body.appendChild(node);
-
-        const node2 = document.createElement("style");
-        node2.innerHTML = blink;
-        document.body.appendChild(node2);
     };
 
     /**
```